src/main/xerus/monstercat/api/Player.kt

```diff
@@ -210,15 +253,26 @@ object Player: FadingHBox(true, targetHeight = 25) {
 	}
 	
 	/** Set the [tracks] as the internal playlist and start playing from the specified [index] */
-	fun playTracks(tracks: List<Track>, index: Int) {
-		playTrack(tracks[index])
-		onFx {
-			if(index > 0)
-				children.add(children.size - 3, buttonWithId("skipback") { playTracks(tracks, index - 1) })
-			if(index < tracks.lastIndex)
-				children.add(children.size - 3, buttonWithId("skip") { playTracks(tracks, index + 1) })
+	fun playTracks(tracks: List<Track>, index: Int = 0) {
+		Playlist.setTracks(tracks)
+		playTrack(Playlist.tracks[index])
```

Comment:
```suggestion
		playFromPlaylist(index)
```

Reply:
Fixed in 0e7760eb51b8e47b3ab5f27c6e12bf7d15193260